src/GraphQL.Tests/Instrumentation/FieldMiddlewareBuilderTests.cs

```diff
@@ -85,7 +86,7 @@ public void can_use_class()
         {
             _builder.Use<SimpleMiddleware>();
 
-            var result = _builder.Build().Invoke(_context).Result;
+            var result = _builder.Build(_context.Schema).Invoke(_context).Result;
```

Comment:
`InvalidOperationException("Schema is null. Schema required for resolving middlewares from DI container.")` if not set